sopel/tools/_events.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import generator_stop
 
 
-class events(object):
+class events:
```

Comment:
Already touched by #2127. Can just drop this change, since it's now an actual `Enum`.